app/UiTPAS/UiTPASServiceProvider.php

```diff
@@ -122,4 +120,25 @@ private function bootstrapEventHandling(): void
         Event::listen(UdbOrganizerApproved::class, [SendMailForUdbOrganizer::class, 'handleUdbOrganizerApproved']);
         Event::listen(UdbOrganizerRejected::class, [SendMailForUdbOrganizer::class, 'handleUdbOrganizerRejected']);
     }
+
+    private function getSmtpDsn(): string
```

Comment:
Can we not just add the full DSN to the config and load it from the `.env` file? So we add a `MAIL_DSN`, which is either the AWS setup or the mailhog setup. If we use that approach, we can even remove the username and password config.

This also makes me wonder if we need another pull request to change from mailhog to mailpit